### DISCUSSION TITLE
Mend Renovate picks up archived repos which it can't manage

### DISCUSSION THREAD

**Discussion by kevin-matthews-vc** [renovatebot/renovate#27619] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

The mend app shows all repos in github regardless if they are archived or not... when onboarding is attempted, no errors occur but it doesn't create a PR because its archived.

IMO it shouldn't even pick up archived or deleted repositories.

I cannot create a reproduction, as its just a setting... if archived it shouldn't pick it up.

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment, marked as answer]
Thanks for the suggestion. Closing as not a bug
